### PR TITLE
update naming for crossplaneio repos and image paths

### PR DESCRIPTION
**Pull request by rathpc**
This builds on https://github.com/crossplaneio/stack-minimal-aws/pull/1 but I don't have rights to push my commit there. Once that PR is merged I will rebase.

### CODE REVIEW

**Comment by hasheddan** [comment]
@rathpc would you mind pinging me once #1 is merged and rebase is ready? I may forget to come back around since I won't get another review request

**Review by jbw976 — APPROVED** [review]

**Pull request merged**
